### ISSUE TITLE
Using modPhred on DRS data

### ISSUE THREAD

**Issue by nemitheasura** [novoalab/modPhred#3]
Hi,

I think your scripts are great and it is cool that you provided a pretrained modification-aware model. I would like to ask some questions regarding modPhred.

I've encountered a few issues while testing your pipeline on my DRS data. Could not solve them by myself, so here I am.

First of all, I cloned the repo and installed all deps according to your instructions. Yet I was unable to run the pipeline (all your scripts at once) with basecalling on-the-fly. Neither using my own data or test dataset pointed out in the docs. The problem is "~/src/modPhred/run" part of the command - I get an error message pointing out that there is no such file or directory. And, actually, there is no file called run or run.py within the repo. I checked multiple times. I ran the command from bash and from python - nothing worked. Am I missing something? 

(my bash version: 5.0.17; my python version: 3.8.3, pyguppyclient version:0.0.6)

Nevertheless, I was able to run the scripts one by one on your prebasecalled test data (~/PRJEB22772/MARC_ZFscreens_R9.4_1D-Ecoli) and yield a proper output. 

So I decided to basecall my own RNA data with guppy 3.5.1 and your pretrained model (rna_r9.4.1_70bps_m6A-m5C-5hmC_hac.cfg). Since I could not do it on the fly, I basecalled my dataset standalone, with --trim_strategy none --reverse_sequence on --u substitution on --fast5_out. Then I used the output as an input in modPhred. 

Unfortunately, guppy_align & mod_report do not work as expected. The produced bamfile contains expected total amount of reads, but - according to samtools flagstat & IGV, none of them is mapped to the reference genome (I have also called the same sample with standard ont model (modification unaware) and the majority of reads were recognized as mapped). In fact, I have tried multiple samples from various organisms and the result was always the same. The entire dataset in bamfile was considered as "unmapped", resulting mod.gz and bedfile were empty, and plot could not be yielded because the script was unable to map the reads. 

Could you provide an example of usage on RNA data and/or the guppy command line? I would appreciate that very much. 

Best,
N.

**Comment by lpryszcz** [comment]
Hi @nemitheasura , 

First of all, thanks for the interest in our work. I've just pushed new release of modPhred and all you issues should be resolved (the ./run link got deleted by mistake in the previous version). 

Regarding direct RNA-sequencing (DRS), there are currently no good guppy models to detect RNA modifications. We have included one in this repo (described in detail in the preprint), but **this model was trained on synthetic molecules in which either all bases are modified or all are unmodified** (more details in supplementary material of the preprint), so it isn't really applicable to biological samples in which only some bases will be modified. 
We'll stress this point more in the new version of the manuscript. 

Hope this clarifies all your concerns :)
